src/coreclr/jit/importercalls.cpp

```diff
@@ -6204,6 +6204,18 @@ void Compiler::impMarkInlineCandidate(GenTree*               callNode,
                                       CORINFO_CALL_INFO*     callInfo,
                                       IL_OFFSET              ilOffset)
 {
+    if (!opts.OptEnabled(CLFLG_INLINING))
+    {
+        /* XXX Mon 8/18/2008
```

Comment:
Do we need the date in the comment here?

Reply:
I don't think so, but I only moved an existing code a bit up, didn't change it